src/python/aqueduct_executor/operators/function_executor/get_extract_path.py

```diff
@@ -14,7 +15,13 @@ def run(spec: FunctionSpec) -> str:
     args = parser.parse_args()
 
     spec_json = base64.b64decode(args.spec)
-    spec = parse_spec(spec_json)
+    try:
+        spec = parse_spec(spec_json)
+    except Exception:
+        # Log the offending spec on failures.
+        print(json.loads(spec_json))
```

Comment:
should we just print the json string here? the spec is wrong so `json.loads` may fail itself

Reply:
ok I printed the raw string here on exception, and printed the nicely formatted json in `parse_spec()`.